### ISSUE TITLE
build shows white page or 404 on github.io

### ISSUE THREAD

**Issue by BexFox** [BexFox/REACT-CRUD#1]
build folder shows just white page or 404 errors.
This project is used by create-react-app.
However, in Netlify, it works well.

Q. How to run github.io with react?

**Comment by Yakup3** [comment]
Hi, you can visit this url (https://github.com/gitname/react-gh-pages) to get some reference.

**Comment by BexFox** [comment]
> Hi, you can visit this url (https://github.com/gitname/react-gh-pages) to get some reference.

I really appreciate your help!

Conclusion, I made it work.
Solution: 
1. Back to the origin master branch
2. Delete build or dist folder & .cache in gh-pages folder.
3. Rewrite "homepage" & build it again (by using npm run build -> npm i gh-pages ->  Add "deploy" -> npm run deploy)

The first few attempts, I followed the guide but it's failed.
I made & changed the branch, edit the private boolean in package.json, and so on.
Because I already pushed the build and .cache in node_modules  to origin master, so web showed white page every time.
